rust/src/lib/unit_tests/route.rs

```diff
@@ -544,3 +544,35 @@ fn test_route_lock_cwnd_is_match() {
     assert!(!desired_route.is_match(&lock_cwnd_not_match_route));
     assert!(desired_route.is_match(&match_route));
 }
+
+#[test]
+fn test_route_without_options_is_match_with_any() {
+    let desired_route: RouteEntry = serde_yaml::from_str(
+        r#"
+        destination: "192.0.2.1"
+        "#,
+    )
+    .unwrap();
+
+    let not_match_route: RouteEntry = serde_yaml::from_str(
+        r#"
+        destination: "192.0.2.100"
+        metric: 1
+        table-id: 2
+        cwnd: 3
+        "#,
+    )
+    .unwrap();
+    let match_route: RouteEntry = serde_yaml::from_str(
+        r#"
+        destination: "192.0.2.1"
+        metric: 1
+        table-id: 2
+        cwnd: 3
```

Comment:
`is_match` is called when deleting the routes, can you also add a unit test matching the cwnd value ? 

```
    let desired_route: RouteEntry = serde_yaml::from_str(
        r#"
        cwnd: 3
        "#,
    )
    .unwrap();
    ```
    assert!(desired_route.is_match(&match_route));